mindsdb/integrations/handlers/iceberg_handler/tests/test_iceberg_handler.py

```diff
@@ -0,0 +1,58 @@
+import unittest
+
+from mindsdb.api.mysql.mysql_proxy.libs.constants.response_type import RESPONSE_TYPE
+from mindsdb.integrations.handlers.iceberg_handler.iceberg_handler import IcebergHandler
+
+
+class IcebergHandlerTest(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        connection_data = {
```

Comment:
can we change this so the values are read from env var

Reply:
The values for the connection data ?